src/ArcanaPrime.sol

```diff
@@ -262,18 +278,19 @@ contract ArcanaPrime is ERC721AQueryable, ERC721ABurnable, Ownable, OperatorFilt
     function mintAspirantList(bytes32[] calldata _merkleProof, uint256 _quantity)
         external
         payable
-        isBelowMaxSupply(_quantity)
+        isBelowMintSupply(_quantity)
         isWhitelisted(_merkleProof, aspirantListMerkleRoot)
         isNotPaused
         isMintOpen(Phases.ASPIRANT)
     {
         uint256 totalPrice = MINT_PRICE * _quantity;
         if (msg.value != totalPrice) revert PriceIncorrect();
 
-        uint256 entitlements = getTotalEntitlements(msg.sender);
-        if (entitlements + _quantity > MAX_ENTITLEMENTS_ALLOWED) revert MaxEntitlementsExceeded();
 
-        _setAux(msg.sender, _getAux(msg.sender) + uint64(_quantity));
+        uint256 totalMints = getTotalMints(msg.sender);
+        if (totalMints + _quantity > MAX_QUANTITY_ALLOWED) revert MaxQuantityAllowedExceeded();
+
+        _setAux(msg.sender, _getAux(msg.sender) + uint64(_quantity << 2));
```

Comment:
This change is done in `mintAspirantList` as well as in `mintAllianceList`, while it was working this way already in `mintPublic`, but why is it not done in `mintArcanaList` ?

Reply:
There's been some changes to the quantities allowed at each phase. The new requirements are as follows. 

1. Max 2 for Arcana Phase
2. Max 3 shared across Aspirant, Alliance and Public